libtransmission/peer-msgs.cc

```diff
@@ -178,7 +178,7 @@ auto constexpr KeepaliveIntervalSecs = time_t{ 100 };
 
 auto constexpr MetadataReqQ = size_t{ 64U };
 
-auto constexpr ReqQ = 512;
+auto constexpr ReqQDefault = 500U;
```

Comment:
add comment explaining where this number comes from, and distinguish client vs peer?